Error on Windows installation

Original report by @AnneshaChowdhury: https://github.com/src-d/tmsc/issues/10

> Hi,
> I am having a hard time with the installation of tmsc. I have been trying to get tmsc up but have been running into some problems. I am installing inside 'venv' on Windows 10.
> This is the snippet of the error I have been getting. 
```cl : Command line warning D9002 : ignoring unknown option '-std=c++11'
    pyuast.cc
    bblfsh/pyuast.cc(481): error C2059: syntax error: '.'
    bblfsh/pyuast.cc(502): error C2143: syntax error: missing ';' before '}'
    bblfsh/pyuast.cc(504): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
    bblfsh/pyuast.cc(504): error C2040: 'ctx': 'int' differs in levels of indirection from 'Uast *'
    bblfsh/pyuast.cc(504): error C2065: 'iface': undeclared identifier
    bblfsh/pyuast.cc(505): error C2059: syntax error: 'return'
    bblfsh/pyuast.cc(506): error C2059: syntax error: '}'
    bblfsh/pyuast.cc(506): error C2143: syntax error: missing ';' before '}'
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\VC\\Tools\\MSVC\\14.11.25503\\bin\\HostX86\\x86\\cl.exe' failed with exit status 2

    ----------------------------------------
Command "c:\progra~1\python36\venv~1\scripts\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\Annesha\\AppData\\Local\\Temp\\pip-install-ur9kqskb\\bblfsh\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\Annesha\AppData\Local\Temp\pip-record-jp280ri4\install-record.txt --single-version-externally-managed --compile --install-headers c:\progra~1\python36\venv~1\include\site\python3.6\bblfsh" failed with error code 1 in C:\Users\Annesha\AppData\Local\Temp\pip-install-ur9kqskb\bblfsh\c\ 
```
> Any help is appreciated!

--- a/bblfsh/pyuast.cc
+++ b/bblfsh/pyuast.cc
@@ -477,29 +477,28 @@ static struct PyModuleDef module_def = {
 PyMODINIT_FUNC
 PyInit_pyuast(void)
 {
-  NodeIface iface = {
-    .InternalType = InternalType,
-    .Token = Token,
-    .ChildrenSize = ChildrenSize,
-    .ChildAt = ChildAt,
-    .RolesSize = RolesSize,
-    .RoleAt = RoleAt,
-    .PropertiesSize = PropertiesSize,
-    .PropertyKeyAt = PropertyKeyAt,
-    .PropertyValueAt = PropertyValueAt,
-    .HasStartOffset = HasStartOffset,
-    .StartOffset = StartOffset,
-    .HasStartLine = HasStartLine,
-    .StartLine = StartLine,
-    .HasStartCol = HasStartCol,
-    .StartCol = StartCol,
-    .HasEndOffset = HasEndOffset,
-    .EndOffset = EndOffset,
-    .HasEndLine = HasEndLine,
-    .EndLine = EndLine,
-    .HasEndCol = HasEndCol,
-    .EndCol = EndCol,
-  };
+  NodeIface iface;
+  iface.InternalType = InternalType;
+  iface.Token = Token;
+  iface.ChildrenSize = ChildrenSize;
+  iface.ChildAt = ChildAt;
+  iface.RolesSize = RolesSize;
+  iface.RoleAt = RoleAt;
+  iface.PropertiesSize = PropertiesSize;
+  iface.PropertyKeyAt = PropertyKeyAt;
+  iface.PropertyValueAt = PropertyValueAt;
+  iface.HasStartOffset = HasStartOffset;
+  iface.StartOffset = StartOffset;
+  iface.HasStartLine = HasStartLine;
+  iface.StartLine = StartLine;
+  iface.HasStartCol = HasStartCol;
+  iface.StartCol = StartCol;
+  iface.HasEndOffset = HasEndOffset;
+  iface.EndOffset = EndOffset;
+  iface.HasEndLine = HasEndLine;
+  iface.EndLine = EndLine;
+  iface.HasEndCol = HasEndCol;
+  iface.EndCol = EndCol;
 
   ctx = UastNew(iface);
   return PyModule_Create(&module_def);

--- a/setup.py
+++ b/setup.py
@@ -11,7 +11,7 @@ from setuptools import setup, find_packages, Extension
 from setuptools.command.build_ext import build_ext
 
 VERSION = "2.10.1"
-LIBUAST_VERSION = "v1.9.1"
+LIBUAST_VERSION = "v1.9.4"
 SDK_VERSION = "v1.16.1"
 SDK_MAJOR = SDK_VERSION.split('.')[0]
 FORMAT_ARGS = globals()

--- a/setup.py
+++ b/setup.py
@@ -10,7 +10,7 @@ from urllib.request import urlopen
 from setuptools import setup, find_packages, Extension
 from setuptools.command.build_ext import build_ext
 
-VERSION = "2.10.0"
+VERSION = "2.10.1"
 LIBUAST_VERSION = "v1.9.1"
 SDK_VERSION = "v1.16.1"
 SDK_MAJOR = SDK_VERSION.split('.')[0]

